yarn-project/noir-contracts/package.json

```diff
@@ -3,23 +3,21 @@
   "version": "0.1.0",
   "type": "module",
   "exports": {
-    "./artifacts": "./dest/artifacts/index.js",
-    "./types": "./dest/types/index.js"
+    ".": "./dest/index.js",
+    "./*": "./dest/*.js"
   },
   "scripts": {
-    "build": "yarn clean && tsc -b",
+    "build": "yarn clean && yarn build:contracts && tsc -b",
     "build:dev": "tsc -b --watch",
-    "clean": "rm -rf ./dest .tsbuildinfo",
+    "clean": "rm -rf ./dest .tsbuildinfo ./src ./target",
     "formatting": "run -T prettier --check ./src && run -T eslint ./src",
     "formatting:fix": "run -T eslint --fix ./src && run -T prettier -w ./src",
     "test": "NODE_NO_WARNINGS=1 node --experimental-vm-modules $(yarn bin jest) --passWithNoTests",
-    "noir:clean": "rm -rf ./src/artifacts ./src/types ../aztec.js/src/account_contract/artifacts ./target/ && mkdir -p ../aztec.js/src/account_contract/artifacts",
-    "noir:build": "./src/scripts/compile.sh",
```

Comment:
It'd be nice to retain a way to compile just a single contract. We have enough contracts that rebuilding everything is significantly slower than rebuilding a single one.

Reply:
Agreed. Tiny script change solves that. Will solve in subsequent PR.